src/MaIN.Infrastructure/Models/ChatDocument.cs

```diff
@@ -18,4 +18,5 @@ public class ChatDocument
     public bool Translate { get; init; }
     public InferenceParamsDocument? InferenceParams { get; init; }
     public MemoryParamsDocument? MemoryParams { get; init; }
+    public object? ConvState { get; init; }
```

Comment:
No explicit type here?

Reply:
Conversation.State given by LLamaSharp is abstract type, therefore cannot be serialized by itself, we can create our own object type individually for that but atm maybe we dont have enough reason for that imo, atm its object because its easy to store it that way :p